### ISSUE TITLE
About 'datasets/proposal_dict.json'

### ISSUE THREAD

**Issue by JaringAu** [ElementAI/wise_ils#2]
Hi,

This is a very insteresting work,
But we are confused about the 'datasets/proposal_dict.json' part. 
We refer to MCG (https://www2.eecs.berkeley.edu/Research/Projects/CS/vision/grouping/mcg/) or COB (http://www.vision.ee.ethz.ch/~cvlsegmentation/cob/code.html), but their proposals are slightly different from the proposals in datasets/proposal_dict.json.
Can you show us more details about how to get the proposals in datasets/proposal_dict.json?
I will be appreciated for your reply as soon as possible, thank you very much!

**Comment by IssamLaradji** [comment]
Thanks, these proposals are from the deepmask repo: `https://github.com/facebookresearch/deepmask`

**Comment by JaringAu** [comment]
Thanks a lot,
but we notice that deepmask need the pixel-level labels in the training phase,
and the COB also need the pixelwise groundtruth.
So in the weakly supervised setting, should we use MCG to generate proposals rather than COB or Deepmask? Could you share your opinion with us?

**Comment by IssamLaradji** [comment]
MCG also requires the pixelwise groundtruth, so they all need pixel-level labels,
(see https://github.com/jponttuset/mcg/issues/11)

However, the weak supervision comes from the fact that they learn from class-agnostic masks and possibly on a different training set.

For proposals that do not require training, you can use selective search.

**Comment by IssamLaradji** [comment]
If you are looking for instance segmentation with even weaker supervision, I recommend this work: https://github.com/jiwoon-ahn/irn which has similar scores as this method, but uses no proposals!

**Comment by JaringAu** [comment]
That makes sense,
appreciate for your help:)

and yeah, we have read jiwoon's paper, it's another insteresting and solid work.

We are also confused about the PRM-points, but it's not related to this issue, so i will close this issue and create a new one.

thanks again for your kindly help.
